src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py

```diff
@@ -232,21 +236,53 @@ def disable_autologging(self) -> None:
             xgboost,
         )
 
+        try:
```

Comment:
I suppose we could use the string to load the module via importlib but tbh seems like overoptimization. LGTM for now if none of the other team members object @safoinme @stefannica @avishniakov